www/src/components/docs-table-of-contents.js

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import { Link } from "gatsby"
+import {
+  fontSizes,
+  colors,
+  zIndices,
+  shadows,
+  space,
+  mediaQueries,
+} from "../utils/presets"
+
+function createItems(items, location) {
+  return (
+    items &&
+    items.map(item => (
+      <li>
+        <Link to={location.pathname + item.url}>{item.title}</Link>
+        {item.items && (
+          <ul css={{ marginLeft: space[6] }}>
+            {createItems(item.items, location)}
+          </ul>
+        )}
+      </li>
+    ))
+  )
+}
+
+function TableOfContents({ page, location }) {
+  return page.tableOfContents.items ? (
+    <div
+      css={{
+        background: colors.white,
+        zIndex: zIndices.feedbackWidget,
```

Comment:
I think this might not be needed, if the shadow is taken off and it sits flat on the page, it'd simplify against situations like this:

![image](https://user-images.githubusercontent.com/21114044/60444618-cdd56180-9bda-11e9-94d2-88a26aace405.png)

where the feedback widget and it overlap.

```suggestion
```

Reply:
:ok